src/docs/exampleComponents/ComposedChart/BandedChart.js

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react';
+import {
+  ComposedChart,
+  Line,
+  Area,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  ResponsiveContainer,
+} from 'recharts';
+
+const data = [
+    {
+      name: "Page A",
+      a: [0, 0],
+      b: 0,
+    },
+    {
+      name: "Page B",
+      a: [50, 300],
+      b: 106,
+    },
+    {
+      name: "Page C",
+      a: [150, 423],
+    },
+    {
+      name: "Page D",
+      b: 312,
+    },
+    {
+      name: "Page E",
+      a: [367, 678],
+      b: 451,
+    },
+    {
+      name: "Page F",
+      a: [305, 821],
+      b: 623,
+    },
+  ];
+  
+  export default class Example extends PureComponent {
+    renderTooltipWithoutRange = ({ payload, content, ...rest }) => {
```

Comment:
there should be a link to codesandbox. but if I created it - it would be under my account.

what should I do here?

Reply:
Create it and link it in the PR comments, I can fork it to the recharts workspace and you can add it before we merge

Reply:
https://codesandbox.io/p/sandbox/simple-area-chart-forked-hncq3r